client/src/components/EditProfile/ProfilePhoto/UploadPhoto.tsx

```diff
@@ -22,18 +24,22 @@ const UploadPhoto: React.FC = (): JSX.Element => {
       formData.append('files', selectedImage);
       formData.append('upload_preset', 'team-reindeer');
 
-      try {
-        await fetch('/upload-images', {
-          method: 'POST',
-          body: formData,
-          credentials: 'include',
-        });
-      } catch (error) {
-        console.log(error);
-      }
+      upload().then((data) => {
```

Comment:
Here, we should pass the `formData` we just created to `upload` so the api call can access the file the user just uploaded.